tools/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+load("@rules_jvm_external//:defs.bzl", "artifact")
+
+java_plugin(
```

Comment:
for java plugin, we should use plugin load.
`load("@rules_java//java:defs.bzl", "java_plugin")`

Reply:
Fixed.